src/app/src/main/kotlin/org/imperial/mrc/hint/controllers/HomeController.kt

```diff
@@ -3,14 +3,15 @@ package org.imperial.mrc.hint.controllers
 import com.github.kittinunf.fuel.core.Headers
 import com.github.kittinunf.fuel.httpGet
 import com.github.kittinunf.fuel.httpPost
+import org.imperial.mrc.hint.AppProperties
 import org.springframework.stereotype.Controller
 import org.springframework.ui.Model
 import org.springframework.ui.set
 import org.springframework.web.bind.annotation.*
 import org.springframework.web.multipart.MultipartFile
 
 @Controller
-class HomeController {
+class HomeController(private val appProperties: AppProperties) {
```

Comment:
Should this be here?

Reply:
Ah maybe you intended to use the appTitle in the index model? https://github.com/mrc-ide/hint/pull/22/files#diff-3a2cf5133208abf39d9bcedbbbbf4a1aR17

Reply:
yes, I think you're right, must have got distracted before I actually updated the model code!